docker-compose.yml

```diff
@@ -28,5 +28,22 @@ services:
     networks:
       - intranet
 
+  alloydbomni:
+    image: google/alloydbomni
+    restart: always
+    ports:
+      - "5432:5432"
```

Comment:
The container doesn't seem to be running. Are there fixes needed to the container? 

```
       	            	connecting to postgres: connecting to database: failed to connect to `user=moov database=moov`:
        	            		127.0.0.1:5432 (localhost): dial error: dial tcp 127.0.0.1:5432: connect: connection refused
        	            		[::1]:5432 (localhost): dial error: dial tcp [::1]:5432: connect: connection refused
```

https://github.com/moov-io/base/actions/runs/11108406519/job/30861274433?pr=437

Reply:
Yea I noticed, not sure why yet. It works locally on macOS. The `alloydbomni` image is known to not work with with Orbstack (which is true locally), but the runner should be using docker anyway. Also tried with postgres image, which fails with same problem. Will need to keep debugging.